demo_analyses/plot_scanpath.py

```diff
@@ -3,15 +3,27 @@
 Created on Mon Sep  5 11:06:43 2022
 
 @author: Marcus
+@contributer: Bob
 """
 
 import pandas as pd
 from pathlib import Path
-import matplotlib.pyplot as plt
 import os
-from psychopy import visual, core
+import cv2
```

Comment:
I am not sure we would want to add a hard dependency on opencv. Perhaps you can try to import cv2 and if you get a ModuleSomethingError fall back to the old implementation (and emit a warning telling the user they get better figures if they pip install opencv-headless or opencv?)

Reply:
I think it would be great to add the option to use opencv! However, I agree with Dee that it would be good to also keep the old implementation, even though drawing is really slow. 
Would of course appreciate a pull request including both options!

Will see if I can find a way to speed up the implementation based on psychopy as well.

Reply:
I integrated both options in the last commit but it might need some polishing. It will now open and close a window for each image which is a bit more overhead and less user friendly.

Reply:
Better then to open the window once under `if use_cv2` and pass it to the function.